deploy/bin/rollback-worker-types.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const utils = require('./worker_type_utils');
```

Comment:
I believe `import foo from 'bar';` is the way of the future for imports.  Eli can correct me if I'm wrong.

Reply:
I am using this syntax to be node 8 compatible.

Reply:
hrm, perhaps that's the problem.  docker-worker is node 6 compatible but we are introducing node 8 for the management/deployment side of things.  Are we also moving the worker code base to be node 8 compatible?

Reply:
That's the plan